### ISSUE TITLE
It gives me great joy that you still use this!

### ISSUE THREAD

**Issue by vsoch** [bissettp/bissettp.github.io#1]
:lizard: <-- happy dinosaur

(feel free to close this, just wanted to say hello! )

:hankey:

**Comment by bissettp** [comment]
I'm sorry that I didn't see this until now. I actually *still* use this in 2025! 

It's good to hear from you. I hope that you're doing well.

**Comment by vsoch** [comment]
Yeah! Thriving! Work is really great at the lab, and this weekend [I was Batman](https://youtu.be/on8UcrpMy8c?si=nrCITjZBVo1z4-Me). I mean, it doesn't really get better than that. ❤️
